serverless/src/functions/builds/handler.ts

```diff
@@ -11,6 +12,14 @@ export const submitBuildInfo = async (): Promise<APIGatewayProxyResult> => {
       createdAt: new Date().toISOString(),
     };
 
+    nfaContract.methods
+      .setTokenBuild(1, 'hash', 'repo')
+      .call((err: string | undefined, res: any) => {
+        if (err) throw new Error(err);
+        console.log('result');
+        console.log(res);
+      });
```

Comment:
I wonder how the gas is going to be paid and also it needs to be signed.

Reply:
this is a placeholder.